StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetAppearance.swift

```diff
@@ -306,5 +308,18 @@ public extension PaymentSheet {
                 public var spacing: CGFloat = 12.0
             }
         }
+        var analyticPayload: [String: Any] {
+            var payload = [String: Any]()
+            payload["style"] = style != PaymentSheet.Appearance.EmbeddedPaymentElement.default.style
+            payload["row"] = row != PaymentSheet.Appearance.EmbeddedPaymentElement.default.row
+            // Convenience payload item to make querying high level appearance usage easier
+            payload["usage"] = payload.values.contains(where: { value in
+                if let boolValue = value as? Bool {
+                    return boolValue == true
+                }
+                return false
+            })
```

Comment:
Do we need this?

Reply:
I thought it was a common pattern we were using -- happy to remove if we don't want it.

Reply:
I think the only place we use this is for top level appearance stuff so we can easily answer "Is appearance being used at all". Peaceful to remove this